test/test_hip_rdna3.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+import unittest
+from tinygrad import Tensor
+from tinygrad.helpers import dtypes, getenv
+from tinygrad.ops import LoadOps
+from tinygrad.codegen.kernel import LinearizerOptions
+from tinygrad.codegen.linearizer import Linearizer
+from tinygrad.renderer.hip import HIPRenderer
+from examples.beautiful_mnist import Model as MNIST
+from examples.hlb_cifar10 import SpeedyResNet
+import extra.hip_wrapper as hip
+
+
+def check_rdna3_compilation(result:Tensor):
```

Comment:
This should not be copied code, it should use all the tinygrad stuff, just be okay with not running the kernel